app/message-attachments/client/messageAttachment.js

```diff
@@ -41,15 +41,8 @@ Template.messageAttachment.helpers({
 		return true;
 	},
 	uploading() {
-		return Template.parentData().msg.uploads;
+		return Template.parentData(2).msg.uploads;
```

Comment:
Why '2' here? FYI in the upstream whole uploading function is removed

Reply:
Probably upstream has added a template between message and message attachment. To access message data in messageAttachment.js, like here https://github.com/WideChat/Rocket.Chat/blob/f92f50cedbe457dfb723cc04122d29e95362e671/app/message-attachments/client/messageAttachment.js#L84

Reply:
Cool. Looks good. May be in the upstream catch up it will be updated